### PR TITLE
Allow 0 in option wizard as valid input

### LINKED ISSUE #4568: It's impossible to make an option with label "0" in the Form Editor
### Affected version(s)

4.13.2

### Description

In the select form field, the option with label "0" is deleted on save:

Screenshot 2022-04-29 at 11-24-10 Formulare Teamanmeldung DE Challenge Felder Formularfeld ID 648 bearbeiten relaunch2022 hands-on-technology org

Screenshot 2022-04-29 at 11-24-40 Formulare Teamanmeldung DE Challenge Felder Formularfeld ID 648 bearbeiten relaunch2022 hands-on-technology org

### PR DESCRIPTION
**Pull request by bezin**
Fixes #4568

### CODE REVIEW

**Review by ausi — DISMISSED** [review]

**Comment by ausi** [comment]
Is Contao 4.9 not affected by this?

**Comment by bezin** [comment]
> Is Contao 4.9 not affected by this?

It certainly is. Will rebase.

**Review by ausi — APPROVED** [review]

**Review by m-vo — APPROVED** [review]

File: core-bundle/src/Resources/contao/widgets/OptionWizard.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function validate()
 			foreach ($options as $key=>$option)
 			{
 				// Unset empty rows
-				if (!$option['label'])
+				if (trim($option['label']) === '')
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/widgets/OptionWizard.php:46]: `trim()` will trigger a deprecation if the argument is not a string: `Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated` Should we add a cast or is `$option['label']` always a string?

Comment (bezin) [reply core-bundle/src/Resources/contao/widgets/OptionWizard.php:46]: AFAIK it always is a string, because it comes straight from the request.

Comment (ausi) [reply core-bundle/src/Resources/contao/widgets/OptionWizard.php:46]: Also, `trim()` is already used on line 52.

**Comment by leofeyer** [comment]
Thank you @bezin.

**Comment by leofeyer** [comment]
Thank you @bezin.

**Pull request merged**